src/Presentation/RestSaveMappingsPresenter.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseRDF\Presentation;
+
+use MediaWiki\Rest\Response;
+use MediaWiki\Rest\ResponseFactory;
+use ProfessionalWiki\WikibaseRDF\Application\MappingList;
+use ProfessionalWiki\WikibaseRDF\Application\SaveMappings\SaveMappingsPresenter;
+use ProfessionalWiki\WikibaseRDF\MappingListSerializer;
+use Wikimedia\Message\MessageValue;
+
+class RestSaveMappingsPresenter implements SaveMappingsPresenter {
+
+	private Response $response;
+
+	public function __construct(
+		private ResponseFactory $responseFactory,
+		private MappingListSerializer $mappingListSerializer
+	) {
+	}
+
+	public function presentSuccess(): void {
+		$this->response = $this->responseFactory->createNoContent();
+	}
+
+	public function presentInvalidMappings( MappingList $mappings ): void {
+		$this->response = $this->responseFactory->createLocalizedHttpError(
+			400,
+			MessageValue::new( 'wikibase-rdf-save-mappings-invalid-mappings' ),
+			[ 'invalidMappings' => $this->mappingListSerializer->mappingListToArray( $mappings ) ]
+		);
+	}
+
+	public function presentSaveFailed(): void {
+		$this->response = $this->responseFactory->createLocalizedHttpError(
+			500,
+			MessageValue::new( 'wikibase-rdf-save-mappings-save-failed' )
```

Comment:
I went with a custom message here since I'm not sure if it's going to be useful to show the exception message coming from `EntityContentRepository::setContent()`.
However, it's simple enough to either add the upstream exception message as an extra field in this response, or to use the more generic `responseFactory->createFromException()`.